Core/Core/Dashboard/K5/View/Schedule/K5ScheduleMissingItemsView.swift

```diff
@@ -0,0 +1,98 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2021-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+
+public struct K5ScheduleMissingItemsView: View {
+    @Environment(\.appEnvironment) var env
+    @State private var isOpened: Bool = AppEnvironment.shared.userDefaults?.isMissingItemsSectionOpenOnK5Schedule ?? true
+    @State private var isArrowPointingDown = AppEnvironment.shared.userDefaults?.isMissingItemsSectionOpenOnK5Schedule ?? true
```

Comment:
This feels a bit redundant, so I tried out removing it, at witnessed the chaos :) So I see why this is necessary..